### PR TITLE
New project: An API to build images on Kubernetes

### PR DESCRIPTION
**Pull request by baijum**
Ref. https://github.com/redhat-developer/build

### CODE REVIEW

**Comment by droslean** [comment]
Please take care of the OWNERS file https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/7252/pull-ci-openshift-release-master-owners/14768#1:build-log.txt%3A13

Also the rehearse is failing. If we merge this, the job will be failing in your repo and it will block the merge queue.

**Comment by baijum** [comment]
> Please take care of the OWNERS file https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/7252/pull-ci-openshift-release-master-owners/14768#1:build-log.txt%3A13

I have added the OWNERS file.
 
> Also the rehearse is failing. If we merge this, the job will be failing in your repo and it will block the merge queue.

We don't have tide configured. For now failure is okay. We will fix once this PR is merged.

**Comment by baijum** [comment]
> The following users are mentioned in OWNERS file(s) but are not members of the openshift org.
> 
> Once all users have been added as members of the org, you can trigger verification by writing `/verify-owners` in a comment.
> 
> * isutton
>   
>   * ci-operator/config/redhat-developer/build/OWNERS
>   * ci-operator/jobs/redhat-developer/build/OWNERS

For now, I have removed `isutton`. We will fix it later.

File: ci-operator/config/redhat-developer/build/redhat-developer-build-master__4.4.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.4"
+  namespace: ocp
+tests:
+- as: unit
+  commands: VERBOSE=2 make test
```
Comment (sbose78) [inline ci-operator/config/redhat-developer/build/redhat-developer-build-master__4.4.yaml]: looks good.

**Comment by baijum** [comment]
> > Also the rehearse is failing. If we merge this, the job will be failing in your repo and it will block the merge queue.
> 
> We don't have tide configured. For now failure is okay. We will fix once this PR is merged.

@droslean 
Now rehearsal is also passing. If no other concern, please approve this PR.

**Pull request merged**